test/ygrunner.py

```diff
@@ -239,7 +260,7 @@ def get_addr_and_fund(yg):
 
 @pytest.fixture(scope="module")
 def setup_ygrunner():
-    load_test_config()
+    load_test_config(ln_backend_needed=True)
```

Comment:
remove

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1182/commits/aacc4f53a9efabf7653b36d4fe613b4e8e06da7b